### ISSUE TITLE
Creation of steering targets broken

### ISSUE THREAD

**Issue by jhg03a** [apache/trafficcontrol#3250]
#### Found in: 
3.0.0-9893.dda9a32d.el7

#### Observed Behavior:
When using TP to create a new Client Steering, you get back the following error: `json: cannot unmarshal string into Go struct field TOSteeringTargetV11.value of type uint64` and no target is created.

#### Expected Behavior:
I expect to be able to create a steering target via traffic portal.

#### Notes:
This appears to be a API regression where perl was implicitly converting the type of the value field whereas golang is explicit.  Tested working in traffic_ops-3.0.0-9385.4ba14bb8.el7 which uses mojo.

#### Tested endpoint:
/api/1.4/steering/9/targets

#### Working body:
```
{
    "deliveryServiceId": 9,
    "targetId": 11,
    "typeId": 44,
    "value": 2
}
```

#### Failing Body:
```
{
    "deliveryServiceId": 9,
    "targetId": 11,
    "typeId": 44,
    "value": "2"
}
```

**Comment by rob05c** [comment]
I'm not seeing dda9a32 but this sounds like https://github.com/apache/trafficcontrol/issues/3223

**Comment by mitchell852** [comment]
yeah, i'm pretty sure @rivasj fixed this

**Comment by rivasj** [comment]
I've confirmed that this issue is a duplicate of #3223 and was fixed with #3224
